src/area-chart/__tests__/area-chart-model-utils.test.ts

```diff
@@ -1,15 +1,15 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import { ChartScale, NumericChartScale } from '../../internal/components/cartesian-chart/scales';
-import { ChartModel } from '../model';
+import { ChartScale, NumericChartScale } from '../../../lib/components/internal/components/cartesian-chart/scales';
+import { ChartModel } from '../../../lib/components/area-chart/model';
```

Comment:
These tests were importing the un-vendored source, which does not work anymore after changes.

Moving to the `../lib` imports so they will use the actual vendored code